src/Uno.UI/UI/Xaml/Media/Animation/Animators/GPUFloatValueAnimator.iOSmacOS.cs

```diff
@@ -417,22 +431,66 @@ private NSValue ToCASkewTransform(float angleX, float angleY)
 			return NSValue.FromCATransform3D(CATransform3D.MakeFromAffine(matrix));
 		}
 
+		private void OnCoreWindowVisibilityChanged(CoreWindow sender, VisibilityChangedEventArgs args)
```

Comment:
not sure if there is a way to test alt-tab behaviors with ui/runtime test?

Reply:
With Uno can we just fakely raise that event?

Reply:
it is more to test that animation killed by os when going into background
are resurrected once back to foreground

Reply:
i can simulate that by invoking FinalizeAnimation+OnCoreWindowVisibilityChanged
but it is not exactly what we are testing

Reply:
There's this API: Xamarin.UITest.iOS.iOSApp.SendAppToBackground (and likely the opposite), but we'll need to case the IApp to iOSApp to do it.

Reply:
Or we could add the API to our Uno.UITest APIs